pkg/apis/core/v1alpha1/federatednamespaceplacement_types.go

```diff
@@ -33,7 +33,9 @@ type FederatedNamespacePlacementStatus struct {
 // +genclient
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
-// FederatedNamespacePlacement
+// FederatedNamespacePlacement is not implemented yet and should not be
+// used at this time.
```

Comment:
Why this not implemented? I saw we already have an example https://github.com/kubernetes-sigs/federation-v2/blob/master/example/sample1/federatednamespace-placement.yaml

Also if this intend to be wip, it is better update PR title by adding `WIP` as keyword.

Reply:
Namespace placement is implemented.  It is slated for revision in #75, since currently namespaces are propagated by default and that complicates a selective (not all namespaces intended to be propagated) cluster-scoped deployment of the federation control plane.

Reply:
I see, thanks @marun , so here I think mentioning `FederatedNamespacePlacement is not implemented` may not that accurate, can we explain this in detail and also link #75 in the comment?